docs/setup_cd_pipeline.md

```diff
@@ -53,7 +53,7 @@ Configuration files allows user assign details about the build requirements and
 
 ```yaml
 build:
-  build-stratergy: Dockerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
```

Comment:
Unfortunately, this has to stay as is see here: https://github.com/AICoE/aicoe-ci/issues/88

Reply:
Thanks for pointing this out in-here.